src/app/app.module.ts

```diff
@@ -19,7 +19,7 @@ import { ShellModule } from './shell/shell.module';
     ShellModule,
     AppRoutingModule,
     QuotingRoutingModule,
-    AppNotFoundRoutingModule,
+    AppLastRoutingModule,
```

Comment:
In theory, the new category and product route definitions could also reside in the app-routing, but I figured, it's better to have them somewhere else close together, because the order of those dynamic routes is important.

Currently every product route (with `-sku123-cat456`) would also match the category route, so it must come before the category route.

One could also think of a route that matches nearly everything and if it resides in the app-routing, it will also capture all routes by pages added with schematics or extensions (as they are added at the bottom of those lists.